src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -103,14 +104,14 @@ const myAppConfig = {
 
```

Comment:
Q from Jim - I realize this is likely outside the scope of this review, but the message in line 95 above within the code block - Before calling API,graphql... - isn't particularly clear. Is this something we can update to make clearer for the user?